README.md

```diff
@@ -1,160 +1,128 @@
-    - Version 3.4+  The Nupic Python repository will need to be upgraded as well before this will be useful.
-  Be sure that your Python executable is in the Path environment variable. The Python that is in your default path is the one
-  that will determine which version of Python the extension library will be built for.
-  NOTE: Anaconda Python not supported.
-  Other implementations of Python may not work.
-  Only the standard python from python.org have been tested.
+    - Version 3.4+
+    - Version 2.7
+      + We recommend the latest version of 2.7 where possible, but the system version should be fine.
+      + Python 2 is Not Supported on Windows, use Python 3 instead.
+
+  Be sure that your Python executable is in the Path environment variable.
+  The Python that is in your default path is the one that will determine which
+  version of Python the extension library will be built for.
+  - NOTE: Anaconda Python not supported.
+  - Other implementations of Python may not work.
+  - Only the standard python from python.org have been tested.
 - Python tools: In a command prompt execute the following.
 ```
-  cd to-repository-root
-  python -m pip install --user --upgrade pip setuptools setuptools-scm wheel
-  python -m pip install --no-cache-dir --user -r bindings/py/packaging/requirements.txt
+cd to-repository-root
+python -m pip install --user --upgrade pip setuptools setuptools-scm wheel
+python -m pip install --no-cache-dir --user -r bindings/py/packaging/requirements.txt
 ```
 
-  Be sure you are running the right version of python. Check it with the following command:
+Be sure you are running the right version of python. Check it with the following command:
 ```
-  python --version
+python --version
 ```
 
-### Building from Source
+## Simple Build for Python users (any platform)
 
-Fork or download the HTM-Community Nupic.cpp repository from https://github.com/htm-community/nupic.cpp
+1) At a command prompt, go to the root directory of this repository.
 
-#### Simple Build for Python users (any platform)
+2) Run: `python setup.py install --user --force`
 
-The easiest way to build from source is as follows.
-```
-    cd to-repository-root
-    python setup.py install --user --force
-```
-Note that `--force` option will overwrite any existing files even if they are
-the same version, which is useful when developing the library & bindings.
+   This will build and install everything.
 
-Note that `--user` option will install the extension libaries in ~/.local so
-that you don't need superuser permissions.
-
-This will build everything including the htm.core static library and Python extension libraries and then install them.
-
-After that completes you are all set to run your .py programs which import the extensions:
- * htm.bindings.algorithms
- * htm.bindings.engine_internal
- * htm.bindings.math
- * htm.bindings.encoders
- * htm.bindings.sdr
```

Comment:
> After that completes you are all set to run your .py programs which import the extensions:

Is this information available someplace?  I would think this would be very important for .py developers to know.

Reply:
Yes, that info should still all be there.  I reformatted it into a table using MarkDown syntax.

Also, the code examples should cover how to use the C++ extensions.

Reply:
> that info should still all be there. 

Hmmm, I must be looking in the wrong place, I don't find it in the README.md.  I know the examples is one place to extract it but that seems more difficult than it should be.  

But then again, perhaps those things should be given in the documentation for each of the entry points that can be called from .py code.  A .py programmer would first find a function they want to call and then look for the module to import and the calling arguments for that function.

Reply:
Oops, I was looking at the wrong file.  I added some more details to that section, which should make it easier for python programmers.